vroom/buildifier.vroom

```diff
@@ -52,3 +52,22 @@ The bzl filetype will use the buildifier formatter by default.
   $ )
 
   :set filetype=
+
+Errors are reported using the quickfix list.
+
+  @clear
+  % #<CR>
+  |java_package(])
```

Comment:
Done (I was confused a bit as well), though note that the prevailing style in some of the other test files is for the continuation versions.